cmake_format/doc/README.rst

```diff
@@ -26,8 +26,29 @@ Integrations
 * There is an official `vscode extension`__
 * Someone also created a `sublime plugin`__
 
+pre-commit tool
+~~~~~~~~~~~~~~~
+
+You can use `pre-commit`__ to easily add
+``cmake-format`` as a pre-commit hook in your git repository. Once
+you `have it installed`__, add this to the ``.pre-commit-config.yaml`` in your
+repository:
+
+.. code:: yaml
+
+    repos:
+    -   repo: https://github.com/cheshirekow/cmake_format.git
+        rev: master
```

Comment:
Installing the "latest" seems like a bad idea. `cmake-format` is still cycling a little on formatting and I think it would be unhelpful if formatting is different for two different users depending on when `pre-commit` cloned  `cmake-format`

Reply:
++, pre-commit in fact [discourages using mutable refs including `master`](https://pre-commit.com/#using-the-latest-version-for-a-repository)

Reply:
Yep, very good point. I've updated it to `v0.5.5`